presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import java.util.Optional;
+
+public class ColumnStatistics
+{
+    public static final ColumnStatistics EMPTY_STATISTICS = ColumnStatistics.builder().build();
+
+    private final Optional<StatisticsValue<Long>> dataSize;
+    private final Optional<StatisticsValue<Long>> rowsCount;
+    private final Optional<StatisticsValue<Object>> min;
+    private final Optional<StatisticsValue<Object>> max;
```

Comment:
Prerequisites:
empty table, no stats provided

Steps:
1. insert into x values(2); // min updated, max updated
2. insert into x values(5); // max updated
3. // wait TTL (to invalidate max stats)
4. insert into x values(1); // min updated

Then min is updated by DML but max is not updated unless refresh is triggered (because of TTL).

It's not overengineering. I just claim to leave it as is instead of introducing any fancy mechanism (that would be overengineering). I don't see any value of merging min and max and gave some scenario as a proof there might be many of similar scenarios we don't think of now. And argument "let's merge and see later" is not an argument.
So if there is at least one argument against merging and no argument for merging... Answer is simple.